### PR TITLE
Typo and update to 3.0

### PR DESCRIPTION
**Pull request by FlorianJacta**

### CODE REVIEW

File: docs/knowledge_base/tips/using_tables/index.md [hunk 1]
```diff
@@ -34,9 +34,9 @@ Gui(page=main_md).run()
 The table definition `<|{food_df}|table|>` (a syntax often used in Taipy) has these parts:
 
 1. `{food_df}`: This is a variable that holds data, like what someone ate in a day.
-2. `table`: This is the name of the control.
+2. *table*: This is the name of the control.
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/using_tables/index.md]:
```suggestion
2. `table`: This is control type name.
```

File: docs/knowledge_base/tips/using_tables/index.md [hunk 2]
```diff
@@ -166,8 +166,8 @@ the table's appearance when applying our **highlight-row** style.
 
 ## Modifying Data
 
-Tables offer various properties for modifying data within the table. Notably, the `on_edit`, 
-`on_add`, and `on_delete` properties can receive a user-defined **callback** function. This 
+Tables offer various properties for modifying data within the table. Notably, the *on_edit*, 
+*on_add*, and *on_delete* properties can receive a user-defined **callback** function. This
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/using_tables/index.md]:
```suggestion
*on_add*, and *on_delete* properties can receive user-defined **callback** functions. This 
```

File: docs/knowledge_base/tips/using_tables/index.md [hunk 3]
```diff
@@ -34,9 +34,9 @@ Gui(page=main_md).run()
 The table definition `<|{food_df}|table|>` (a syntax often used in Taipy) has these parts:
 
 1. `{food_df}`: This is a variable that holds data, like what someone ate in a day.
-2. `table`: This is the name of the control.
+2. *table*: This is the name of the control.
 
-In this example, the data in `food_df` is a **pandas DataFrame**. If it works better for what 
+In this example, the data in *food_df* is a **pandas DataFrame**. If it works better for what
```
Comment (FabienLelaquais) [inline docs/knowledge_base/tips/using_tables/index.md:39]: I don't think you need to create a Pandas DataFrame at this point.
A dict of array will do just fine, and simplifies the reading.
That would depend on what you do with this variable elsewhere in the code...

**Review by FabienLelaquais — APPROVED** [review]
Very minor typography comments

File: docs/knowledge_base/tips/using_tables/index.md (hunk 3 above)
Comment (FlorianJacta) [reply docs/knowledge_base/tips/using_tables/index.md:39]: The Pandas Data frame will be useful in the article after.

Comment (FabienLelaquais) [reply docs/knowledge_base/tips/using_tables/index.md:39]: 👌

**Review by jrobinAV — APPROVED** [review]

**Pull request merged**